### PR TITLE
Makes Gamepad example able to be tested with Windows 10/11

### PR DESCRIPTION
**Pull request by SuGlider**
## Description of Change
Initial example code had no effect with Win10/11 because BUTTON_START was not recognized. 
This PR makes it visible in the Windows Game Controller Testing TAB.

## Tests scenarios
ESP32-S2 and ESP32-S3 (OTG mode)

## Related links
https://github.com/espressif/arduino-esp32/issues/6922#issuecomment-1500791745
https://github.com/espressif/arduino-esp32/issues/6922#issuecomment-1501858344

### CODE REVIEW

File: libraries/USB/examples/Gamepad/Gamepad.ino [hunk 1]
```diff
@@ -18,9 +20,14 @@ void setup() {
 
 void loop() {
   int buttonState = digitalRead(buttonPin);
-  if ((buttonState != previousButtonState) && (buttonState == LOW)) {
-    Gamepad.pressButton(BUTTON_START);
-    Gamepad.releaseButton(BUTTON_START);
+  if (buttonState != previousButtonState) {
+    if (buttonState == LOW) { // BOOT Button pressed
+      // changes many states
+      Gamepad.send(-100, 100, 50, 100, -100, -50, HAT_DOWN_RIGHT, BUTTON_TL);
```
Comment (me-no-dev) [inline libraries/USB/examples/Gamepad/Gamepad.ino:26]: last argument here is mask, so correct value would be `(1 << BUTTON_TL)`.

**Comment by SuGlider** [comment]
After testing all combinations, it seems that no button is recognized by Win11.
Maybe the report is different.

**Comment by me-no-dev** [comment]
Does it all work (buttons and axis) when you have that window Active, or only some things are working? Generally that is very strange... usb report is really standard.

**Comment by SuGlider** [comment]
> Does it all work (buttons and axis) when you have that window Active, or only some things are working? Generally that is very strange... usb report is really standard.

All working fine. I'll submit the changes I made to the example that are able to test everything.

**Pull request merged**